packages/babel-preset-env/test/fixtures/corejs2/usage-evaluated-not-confident/output.mjs

```diff
@@ -1,8 +1,5 @@
-import "core-js/modules/es6.array.map.js";
-import "core-js/modules/web.dom.iterable.js";
-import "core-js/modules/es6.array.iterator.js";
-import "core-js/modules/es6.object.to-string.js";
 import "core-js/modules/es7.object.values.js";
+import "core-js/modules/es6.array.map.js";
```

Comment:
I'm not sure because the old code included these polyfills, but the new one seems correct to me.